integration/setup.go

```diff
@@ -0,0 +1,130 @@
+package integration
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/kaspanet/kaspad/app"
+	"github.com/kaspanet/kaspad/config"
+	"github.com/kaspanet/kaspad/dbaccess"
+)
+
+func setup(t *testing.T) (app1, app2, app3 *app.App, client1, client2, client3 *rpcClient, teardownFunc func()) {
+	config1, config2, config3 := configs(t)
+
+	databaseContext1, databaseContext2, databaseContext3 := openDBs(t, config1, config2, config3)
+
+	app1, app2, app3 = newApps(t, config1, config2, config3, databaseContext1, databaseContext2, databaseContext3)
+
+	app1.Start()
+	app2.Start()
+	app3.Start()
+
+	client1, client2, client3 = rpcClients(t)
+
+	return app1, app2, app3, client1, client2, client3,
+		func() { teardown(t, app1, app2, app3, databaseContext1, databaseContext2, databaseContext3) }
+}
+
+func rpcClients(t *testing.T) (client1, client2, client3 *rpcClient) {
```

Comment:
I would remove this function and instead replace it with newAppRPCClient(config, rpcAddress)